Tasks/Leshchev/Task6/Task6.py

```diff
@@ -0,0 +1,72 @@
+
+x = [1, 2, [2, 4, [[7, 8], 4, 6]]]             #Task 6.1  
+s = 0
+
+def summ(x, i):        #Without the cycle for
+    global s
+    if i <= len(x)-1 and isinstance(x[i], int):
+        s += x[i]
+        i += 1
+        summ(x, i)
+        return s
+    elif i <= len(x)-1 and isinstance(x[i], list):
+        summ(x[i], 0)
+        i +=1
+        summ(x, i)
+        return s
+    elif i > len(x):
+        return s
+        
+print(summ(x, 0))
+
+s = 0     
+def summ_2(x):                               #With the cycle for
+	global s
+	for i in x:
+		if isinstance(i, int):
+			s += i
+		if isinstance(i, list):
+			summ_2(i)
+	return s
+
+print(summ_2(x))        
```

Comment:
both are bad options since you're using a global variable, it can be easily done without it